woocommerce/woocommerce
# Remove no-op migration from 8.0.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Relates to https://github.com/woocommerce/woocommerce/pull/37050. After further testing, we've determined that the migration code responsible for finding and removing 'stray' order placeholders is not actually required, and appears not to do anything. On that basis, we've decided to remove it from the 8.0 release.

If we later find circumstances that require it, we can easily re-introduce it and have it target a future release—but for the time being it seemed wiser to avoid an unnecessary migration (and the queries it generates).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

- Code review should be sufficient.
- This removes a migration (a callback running after updating to 8.0), so it doesn't really provide anything to be tested manually. See https://github.com/woocommerce/woocommerce/pull/37050 for further context.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
